package.json

```diff
@@ -91,8 +92,9 @@
     "conventional-changelog-lint-config-angular": "0.4.1",
     "css-loader": "0.23.1",
     "cypress": "2.1.0",
+    "enzyme": "3.3.0",
+    "enzyme-adapter-react-15.4": "1.0.5",
```

Comment:
I have nuked my nodes folder, ran:
nvm install (switches to the right node)
npm install
npm test

![image](https://user-images.githubusercontent.com/10524/38427115-af38b932-39b8-11e8-83aa-8d3238c2effd.png)

All tests failed becauese of that.

Reply:
Forget about it, it was solved.

Reply:
Yeah, ran into the same thing previously 😞